### ISSUE TITLE
Cross Platform Tournaments

### ISSUE THREAD

**Issue by ghost** [djrscally/pubg_reporting#20]
I am currently using your process to create analysis that supports my PUBG tournaments. Cross platform custom matches are now a thing and we had our first tournament on the 18th that included both xbox and PS players.

The current setup only pulls data for Xbox players so none of my PS players are appearing within the analysis. Would be great to develop on the existing setup to pull data from both shards.

Kind regards
MajorClagNut

**Comment by Devastatin** [comment]
run twice... once across each shard?

**Comment by djrscally** [comment]
No. He means he wants to be able to include players from multiple different
shards; Xbox, PlayStation, steam etc. The way I've written the code
initially means you can only do one shard. I'm working on it now, but it
sorta lends itself well to a player management function (add / remove
players from the sync) so it's taking a wee while.

**Comment by djrscally** [comment]
And immediately I realise I'm being stupid and you're suggesting he run the
process twice across both shards. Ignore my previous reply entirely. I
blame lack of coffee.

**Comment by Devastatin** [comment]
Ah, I'm using the code to automatically expand the player population for
metrics to determine statistical outliers.

My issue is re run speed for the same population for areas that never
update.
